addons/website/static/src/components/dialog/seo.js

```diff
@@ -411,7 +411,8 @@ export class OptimizeSEODialog extends Component {
             this.canEditSeo = this.data.can_edit_seo;
             this.canEditDescription = this.canEditSeo && 'website_meta_description' in this.data;
             this.canEditTitle = this.canEditSeo && 'website_meta_title' in this.data;
-            this.canEditUrl = this.canEditSeo && 'seo_name' in this.data;
+            // The URL can be edited only if it contains soe_name or seo_name_default as slug.
```

Comment:
Generally you want comments to explain why we do something, more than what we do
```suggestion
            // The URL can be edited only if it contains seo_name or seo_name_default as slug.
```